### ISSUE TITLE
See-through non-shoot-through sectors display HOM

### ISSUE THREAD

**Issue by chocolate-import** [chocolate-doom/chocolate-doom#63]
> The following bug was [originally reported](https://sourceforge.net/p/chocolate-doom/bugs/63/) on Sourceforge by [never_again](https://sourceforge.net/u/never_again/profile/), 2010-01-09 03:46:00:

Example: OTTAWAU.WAD E1M1 http://www.doomworld.com/idgames/index.php?id=1651 
Sectors 226 and 300 make a transparent but non-shoot-through window that is displayed fine in vanilla but as HOM in Chocolate DOOM v1.2.1. There are several such windows on this level (the bare OTTAWAU.WAD was used, without the sprites/flats install).
I believe the sector trick used here was first described in newtechn.zip http://www.doomworld.com/idgames/index.php?id=401 
Basically it involves two sectors, one completely surrounding the other, with the outer sector's front sidedefs pointing out and back sidedefs removed.

Chocolate DOOM also tends to crash randomly with the player around that area, which could be related to the above. The crash is silent, with no errors reported in stdout.txt (last line: "ST_Init: Init status bar") or stderr.txt (blank).

Example screenshots:
http://s955.photobucket.com/albums/ae34/tpoppins/Windows%20on%20OTTAWAU%20E1M1/
